src/VirtoCommerce.ExperienceApiModule.Core/Middleware/AuthorizationErrorHandlingMiddleware.cs

```diff
@@ -0,0 +1,33 @@
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using VirtoCommerce.ExperienceApiModule.Core.Infrastructure.Authorization;
+
+namespace VirtoCommerce.ExperienceApiModule.Core.Middleware
+{
+    public class AuthorizationErrorHandlingMiddleware
```

Comment:
No interfaces?

Reply:
Middlewares can be registered as convention-based, so no need for interfaces.